firmware/src/transceiver.h

```diff
@@ -254,11 +284,16 @@ void Transceiver_Initialize(const TransceiverHardwareSettings *settings,
 /**
  * @brief Change the operating mode of the transceiver.
  * @param mode the new operating mode.
+ * @param token to token to the TransceiverEventCallback when the mode change
```

Comment:
SPaG I'm not sure exactly what you're trying to say, all I know is it's not right. :)